redcap_bridge/cli.py

```diff
@@ -28,18 +32,29 @@ def main(command_line=None):
                           help="Format to store the data (json/csv)")
     download.add_argument("-c", "--compressed", action='store_true',
                           help="Compress the output file (use labels and merge checkbox columns)")
+    download.add_argument("--server", required=False, type=str, nargs=1, metavar='server',
```

Comment:
It would make sense to also provide a short command here like `-s`, as with the other arguments already implemented.

Reply:
What error?